dependencies.json

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
I agree we need to have a single source of truth for dependencies but should'nt we include some "human" information like a dependency's description or a category. 

Should we add some "human-oriented"information to `vertx-stack`? Some where else ?